### ISSUE TITLE
Can I help maintain

### ISSUE THREAD

**Issue by AverseABFun** [zloirock/core-js#1187]
I won't be able to be on *too* much but I'll be able to be on at least a couple hours per day. I really really really really want to help. I can tell you need it from the readme.

**Comment by zloirock** [comment]
Initially, you could try help with [`help wanted`](https://github.com/zloirock/core-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22) issues -)

**Comment by AverseABFun-windows** [comment]
Will do.

**Comment by luisantoniocrag** [comment]
Same here I will try some `help wanted` issues

**Comment by Bekacru** [comment]
> Initially, you could try help with [`help wanted`](https://github.com/zloirock/core-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22) issues -)

Same here and I saw the first issue. I can build a little doc website for the readme. Should I get started or is there anything I should know?

**Comment by zloirock** [comment]
@Bekacru it will be required, but not right now - initially, I should think about the static site generator and the frame of the site. Now you could help with docs in the repo.

**Comment by iambenkay** [comment]
I will also help contribute by picking up some help wanted issues

**Comment by DreamOfIce** [comment]
Considering the current shortage of core-js maintenance staff, perhaps you could also use the following two options for building documentation, which are heavily used for documentation sites and have many out-of-the-box themes/plugins.
- [Vuepress v2](https://v2.vuepress.vuejs.org) +[ vuepress-theme-hope](https://github.com/vuepress-theme-hope)
- [Vitepress](https://vitepress.vuejs.org/) + default theme
If you are ready to use these two options, I am happy to help or be responsible for building 😃

**Comment by AverseABFun-windows** [comment]
Looks like vitepress is the best option. Vuepress2 is still in beta and
will likely contain breaking changes between minor releases.

**Comment by AverseABFun-windows** [comment]
Never mind, vitepress is still in alpha. How about vuepress v1?

**Comment by zloirock** [comment]
Let's move the discussion about docs improvement to #1053.

**Comment by AverseABFun-windows** [comment]
Yeah

**Comment by DreamOfIce** [comment]
> Never mind, vitepress is still in alpha. How about vuepress v1?

First, Vuepress v1 is no longer maintained: https://github.com/vuepress-theme-hope/vuepress-theme-hope/discussions/1535
Second, v2 is better than v1 in terms of development experience (mainly thanks to the speed of `vite`)

## Regarding the stability
many projects are already using v2, and I personally have been using it for a while and haven't encountered breaking changes.

## Vuepress or Vitepress?
Vuepress-theme-hope provides very comprehensive features, and Vuepress plug-in ecology is also very good, basically can meet all the needs
But I did not find so comprehensive vitepress theme, many features may have to write our own.

So, in summary I personally recommend using Vuepress v2

**Comment by AverseABFun-windows** [comment]
Ok, sounds good. I didn't know that so thanks :).

**Comment by jxintang** [comment]
I would like to try some help wanted issues, too

**Comment by AverseABFun** [comment]
yeah no i was such an idiot and a skid back when i made this i'm closing this issue
i do still support you though @zloirock!

**Comment by AlexDev404** [comment]
@AverseABFun If you supported him, you would be helping in the ways he has already mentioned several times--instead of being like everyone else who already "supports" him
